src/sentry/api/endpoints/codeowners/__init__.py

```diff
@@ -14,8 +14,7 @@
 from sentry.utils import metrics
 
 # Max accepted string length of the CODEOWNERS file
-MAX_RAW_LENGTH = 100_000
-HIGHER_MAX_RAW_LENGTH = 3_000_000
+MAX_RAW_LENGTH = 1_000_000
```

Comment:
Why 1M instead of 3M?

Reply:
I've talked to Brian and we decided to continue to allow the orgs with the feature flag to have length 3M, while everyone else will be allowed to have length 1M. This will allow us to gradually increase the length after we ensure 1M works for all orgs